charts/fluentd-elasticsearch/templates/daemonset.yaml

```diff
@@ -139,13 +139,12 @@ spec:
                 exit 1;
               fi;
               touch -d "${STUCK_THRESHOLD_SECONDS} seconds ago" /tmp/marker-stuck;
-              if [[ -z "$(find /var/log/fluentd-buffers -type f -newer /tmp/marker-stuck -print -quit)" ]];
+              if [ -z "$(find /var/log/fluentd-buffers -type f -newer /tmp/marker-stuck -print -quit)" ];
               then
-                rm -rf /var/log/fluentd-buffers;
```

Comment:
Also removing this line, as fluentd getting stuck is usually _nopt_ dependent on the buffer contents and this will ensure we do not lose data in the event of a restart of fluentd (as long as the buffers don't get full, obviously).

Reply:
^ I've abandoned that change; it could be controversial and I'll keep it to 1 change per PR to keep things clean.